example-project/cbindgen.toml

```diff
@@ -0,0 +1,7 @@
+include_guard = "EXAMPLE_PROJECT_H"
+include_version = true
+language = "C"
+cpp_compat = true
+
+[export.rename]
+"OddCounter" = "ExampleProjectOddCounter"
```

Comment:
Can we always use `OddCounter`?

Reply:
Sure, we could.

But AFAIK it is common for a C API to prefix every public symbol with the project name (to avoid name clashes), so that's what I did here.
Also, I think the `export.rename` feature is really useful, so it's nice to show it here.

But if you don't like it, I can remove it.

Reply:
Seems a good reason, let's leave it as-is